src/main/java/ch/wisv/areafiftylan/products/controller/TicketRestController.java

```diff
@@ -158,6 +160,24 @@ public AvailableTicketsDTO getAvailableTickets() {
         return createResponseEntity(HttpStatus.OK, "TicketType successfully deleted.");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
+    @PostMapping("/types/{typeId}/buyable")
+    public ResponseEntity<?> updateTicketTypeBuyable(@PathVariable Long typeId, @RequestBody @Validated TicketBuyableDTO buyable) {
+        TicketType ticketType = ticketService.getTicketTypeById(typeId);
+        ticketType.setBuyable(buyable.isBuyable());
+        ticketService.updateTicketType(typeId, ticketType);
+        return createResponseEntity(HttpStatus.OK, "TicketType successfully updated", ticketType);
+    }
+
+    @PreAuthorize("hasRole('ADMIN')")
+    @PostMapping("/types/{typeId}/option")
+    public ResponseEntity<?> addOptionToType(@PathVariable Long typeId, @RequestBody @Validated TicketOptionDTO option) {
```

Comment:
This follows REST principles, as you're adding a new object to the Options "collection" of a type :+1: